packages/web/src/components/table/Table.tsx

```diff
@@ -658,7 +662,7 @@ export const Table = ({
       <InfiniteLoader
         isRowLoaded={isRowLoaded}
         loadMoreRows={loadMoreRows}
-        rowCount={totalRowCount}
+        rowCount={totalRowCount == null ? rows.length : totalRowCount}
```

Comment:
in the case where `totalRowCount` is not known I think the InfiniteLoader docs tell you to give an "arbitrarily high number" but just using `rows.length` there works.